### PR TITLE
chore: Adding redis 6.2 eol date.

### PR DESCRIPTION
**Pull request by awais786**
https://redis.io/docs/about/releases/#support

### CODE REVIEW

File: barcalendar.py [hunk 1]
```diff
@@ -440,7 +440,7 @@ def write(self):
 redis_releases = [
     ('5.6', 2020, 4, 2021, 10),
     ('6.0', 2020, 5, 2022, 5),
-    ('6.2', 2021, 8, 2023, 2),
+    ('6.2', 2021, 8, 2024, 4),
     ('7.0', 2022, 4, 2023, 10),
```
Comment (iamsobanjaved) [inline barcalendar.py:444]: Update this as well.

File: barcalendar.py [hunk 2]
```diff
@@ -440,8 +440,8 @@ def write(self):
 redis_releases = [
     ('5.6', 2020, 4, 2021, 10),
     ('6.0', 2020, 5, 2022, 5),
-    ('6.2', 2021, 8, 2023, 2),
-    ('7.0', 2022, 4, 2023, 10),
+    ('6.2', 2021, 8, 2024, 4),
+    ('7.0', 2022, 4, 2024, 10),
```
Comment (nedbat) [inline barcalendar.py:444]: Where did these dates come from? I don't see them on either of the URLs above?

Comment (iamsobanjaved) [reply barcalendar.py:444]: They haven't provided the EOL date anywhere, we just calculated it from the above links. As per [this link](https://redis.io/docs/about/releases/#support), they support the latest version and two previous versions get maintenance and security patches and they release a new version every year. 7.2 isn't yet released, but two release candidates are released https://github.com/redis/redis/releases. So until 8.0 released next year, 6.2 will get security and maintenance patches.

**Review by iamsobanjaved — APPROVED** [review]

**Comment by nedbat** [comment]
Thanks for updating the code.  Next time, it would be better to use a conventional commit message and squash the five commits into one. 

Would you like me to update https://docs.google.com/spreadsheets/d/11DheEtMDGrbA9hsUvZ2SEd4Cc8CaC4mAfoV8SVaLBGI/edit#gid=195838733 or will you be doing that?

**Pull request merged**